sematic/ui/src/components/MenuPanel.tsx

```diff
@@ -9,18 +9,16 @@ import {
   useTheme,
```

Comment:
Typo in the PR description: `resoltuion` -> `resolution`.

Reply:
Keen eyes! Thanks! 

Corrected.